lib/vtk/analytics.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'json'
+require 'net/http'
+require 'open-uri'
+require 'uri'
+
+module Vtk
+  # Provides command analytics to VTK team
+  class Analytics
+    CLIENT_KEY = '456f76a9dc1a235690017dab981af904'
+
+    attr_reader :name, :title, :hostname
+
+    def initialize(name:, title:, hostname: nil)
+      @name = "vtk.#{name}"
+      @title = "VTK #{title}"
+      @hostname = hostname || `hostname -f`.chomp
+    end
+
+    def log
+      return if ENV['CI'] || ENV['TEST']
+
+      Process.fork do
+        begin
+          exit unless internet?
+
+          emit_event
+        rescue StandardError
```

Comment:
Also can StandardError be removed? I think StandardError is assumed if nothing is provided. (You could argue that this reads better / is easier to follow though)

Reply:
StandardError is assumed but by convention we don't use implicit rescue. It's basically saying, "Hey I'm rescuing everything and I meant to. I'm not being lazy, I really want to rescue everything."